nuxt.config.js

```diff
@@ -1,13 +1,15 @@
-import colors from 'vuetify/es5/util/colors'
+import { redirect } from '@nuxtjs/auth/lib/module/defaults';
+import colors from 'vuetify/es5/util/colors';
+require("dotenv").config();
 
 export default {
   // Disable server-side rendering: https://go.nuxtjs.dev/ssr-mode
   ssr: false,
 
   // Global page headers: https://go.nuxtjs.dev/config-head
   head: {
-    titleTemplate: '%s - test-blueprint',
-    title: 'test-blueprint',
+    titleTemplate: '%s - NuxT',
```

Comment:
why change the title ?

Reply:
forgot to change it back, i alrady change it back